index.js

```diff
@@ -86,6 +86,6 @@ function fastifyGracefulShutdown(fastify, opts, next) {
 }
 
 module.exports = fp(fastifyGracefulShutdown, {
-  fastify: '>=3.0.0',
+  fastify: '>=5.0.0',
```

Comment:
fastify 5 requires fastify-plugin 5, which is incompatible with older fastify versions